app/models/ability.rb

```diff
@@ -10,6 +10,10 @@ def initialize(user)
       can [:update], Card do |card|
         card.owned_by?(user)
       end
+      can [:read], User
+      can [:update], User do |userr|
+        userr == user
```

Comment:
So this means only the current user can update cards? If that is the case I would prefer something a little clearer `userr => session_user`, `current_user`, `logged_in_user`. Again just to help the reader.